scripts/core/editor3/reducers/editor3.jsx

```diff
@@ -81,7 +81,9 @@ export const onChange = (state, newState, force = false) => {
     let contentChanged = state.editorState.getCurrentContent() !== newState.getCurrentContent();
 
     if (contentChanged || force) {
-        state.onChangeValue(editorState.getCurrentContent());
+        const plainText = (state.editorFormat || []).length === 0;
```

Comment:
I might confused you with my suggestions, but in this case it's better to use `plainText = showToolbar === false`, because otherwise you are duplicating the logic from here https://github.com/superdesk/superdesk-client-core/blob/787408ceadd538fb26d1694edc5d5bd144b8e3e4/scripts/core/editor3/store/index.js#L44

You also don't take `singleLine` into account. Let's keep the logic in one place by using `plainText = showToolbar === false`. Once you have the logic in one place you can fix the naming and remove `showToolbar` everywhere replacing it with `plainText === false`. If you want, you can skip the second part and I can do it sometime.

Reply:
I agree with having the logic in one place but I still think I shouldn't go back to `plainText = showToolbar`, as you said we might want to hide the toolbar at some point even on formatted editors. I think it would be better to export `!props.singleLine && (props.editorFormat || []).length > 0;` as `isEditorPlainText` function or something similar and use that.

Reply:
I agree, let's do that :)